phy/plot/view_models/base.py

```diff
@@ -182,6 +183,12 @@ def on_close(self):
         self._view.visual.spike_clusters = []
         self._view.update()
 
+    def on_key_press(self, event):
+        if event.key == 'h' and 'control' not in event.modifiers:
```

Comment:
"not in event.modifiers"?... Doesn't that contradict https://github.com/kwikteam/phy/pull/284/files#diff-d35c2c3902bbcd7194cb3cd4461b26b8R53?